src/composables/useConversationInfo.ts

```diff
@@ -56,32 +64,32 @@ export function useConversationInfo({
 			return ''
 		}
 
-		const params = item.value?.lastMessage.messageParameters
-		let subtitle = (getMessageIcon(item.value?.lastMessage) + ' ' + escapeHtml(item.value?.lastMessage.message)).trim()
+		const params = item.value.lastMessage!.messageParameters
```

Comment:
General TS note:

We can define a new variable for the last message. Then we don't need to add `!` in all the places where we use it, and there is a place for a comment to explain, why `!` is added here.

```ts
/* Without reactivity */

// lastMessage is guaranteed to be defined becase <proof>
const lastMessage = item.value.lastMessage!

/* With reactivity */

// lastMessage is guaranteed to be defined becase <proof>
const lastMessage = toRef(() => item.value.lastMessage!)
```

Reply:
`vueuse` provide `toRef` for compatibility

Reply:
`computed` is also ok, it's just slightly slower (on nano-optimization level) and has a different purpose semantically (lazy derived computing).

Reply:
I haven't tried, but wouldn't that yield an error on composable call?

As computed is lazy, it won't be called until we asked for it (which we only do after ensuring lastMessage exists)